apps/ui/src/Heading.tsx

```diff
@@ -37,6 +37,7 @@ const headingStyles = cva([], {
 			'steel-darker': 'text-steel-darker',
 			hero: 'text-hero',
 			'hero-dark': 'text-hero-dark',
+			'hero-darkest-80': 'text-hero-darkest/80',
```

Comment:
instead of this, I think it might be better to introduce a prop `opacity`, that way we do not need to have a style prop for each color + opacity combination. Otherwise, we could also wrap the `Heading` component with a div and just put `text-opacity-80` so it trickles down also.

```
opacity: {
  80: 'text-opacity-80'
}
```